scripts/rollup/shims/facebook-www/ReactErrorUtils.js

```diff
@@ -1,19 +0,0 @@
-/**
- * Copyright 2013-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @providesModule ReactErrorUtils
```

Comment:
We don't use the shim on www, we reference it through the secret internal object directly.